src/main/java/com/faforever/gw/model/SolarSystem.java

```diff
@@ -89,4 +89,17 @@ public Faction uniqueOwner() {
 
         return uniqueFaction;
     }
+
+    @JsonIgnore
+    public boolean isReachable(Faction faction) {
+
+        for (SolarSystem system : getConnectedSystems()) {
```

Comment:
please add a note above that this approach only works with bidirectional links (just so we don't forget it later)